### PR TITLE
cherry pick for issue ansible-collections/azure#161  from ansible-collections/azure for v2.9

### PR DESCRIPTION
**Pull request by ShwetaliBerad**
##### SUMMARY
updated code to fetch availability zone info into hostvars via Azure Inventory Script
https://github.com/ansible-collections/azure/issues/161

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
Dynamic Inventory Script

### CODE REVIEW

**Comment by ShwetaliBerad** [comment]
@haiyuazhang @Fred-sun PR for v2.9 as per your request for issue https://github.com/ansible-collections/azure/issues/161
could you please review it ?

**Comment by relrod** [comment]
Please add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) for this backport.

**Comment by ShwetaliBerad** [comment]
@relrod Please check updated change log

File: changelogs/fragments/161-ansible-collection-azure-inventory-fixes.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bugfixes:
+- azure_rm invnetory - inventory plugin update to fetch availablity zone information of VM in hostvars.
```
Comment (relrod) [inline changelogs/fragments/161-ansible-collection-azure-inventory-fixes.yaml]:
```suggestion
- azure_rm inventory plugin - update to fetch availability zone information of VM in hostvars.
```

**Pull request merged**